src/chains/ethereum/ethereum/src/forking/handlers/ws-handler.ts

```diff
@@ -98,7 +98,7 @@ export class WsHandler extends BaseHandler implements Handler {
     }
   }
 
-  private connect(connection: WebSocket) {
+  private connect(connection: WebSocket, options: EthereumInternalOptions) {
```

Comment:
Can we require just the `LoggingConfig` here rather than the entire config object?

Reply:
I'm glad you have an opinion because I kept going back and forth on this. Fixed.

Though the types here are a bit confusing - I think I got it right with `EthereumInternalOptions["logging"]` rather than `LoggingConfig`.